src/lsp/cobol_parser/grammar.mly

```diff
@@ -1631,6 +1631,10 @@ let data_type_clause := TYPE; TO?; ~ = name; < >
 let usage_clause :=
```

Comment:
```suggestion
let usage_clause [@recovery Binary] [@symbol "<usage-clause>"] :=
```
For later: we need to add a `dummy_usage_clause` somewhere in `Cobol_ptree.Dummies` (so we could check later that some nodes have been automatically "guessed").

Reply:
Those annotations are sometimes hard to put in the right place. I'll push a commit that moves them to `usage`.